.github/workflows/close-stale-feature-requests.yml

```
@@ -15,14 +15,15 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@v5
```

The current version is `v6`, it sets `not_planned` by default and is otherwise identical to `v5`. I suggest to use the latest version when touching this anyway:
```suggestion
      - uses: actions/stale@v6
```

---

Yeah, I've noticed that, but wasn't sure if v6 might bring more new issues than v5 (which would be required). I can update, but perhaps we want to do this piecemeal.

---

The difference is literally just the default: https://github.com/actions/stale/compare/v5...v6

---

Ah, thanks! [Updated](0b8d0557d0fcc6dfbdd4d6fb4c2a1a137237cb46).